src/sinks/aws_cloudwatch_metrics.rs

```diff
@@ -267,55 +284,63 @@ impl Service<PartitionInnerBuffer<Vec<Metric>, String>> for CloudWatchMetricsSvc
             return future::ok(()).boxed();
         }
 
-        let input = PutMetricDataInput {
-            metric_data,
-            namespace,
-        };
-
-        debug!(message = "Sending data.", input = ?input);
         let client = self.client.clone();
-        Box::pin(async move { client.put_metric_data(input).await })
+
+        Box::pin(async move {
+            client
+                .put_metric_data()
+                .namespace(namespace)
+                .set_metric_data(Some(metric_data))
+                .send()
+                .await?;
+            Ok(())
+        })
     }
 }
 
 #[derive(Debug, Clone)]
 struct CloudWatchMetricsRetryLogic;
 
 impl RetryLogic for CloudWatchMetricsRetryLogic {
-    type Error = RusotoError<PutMetricDataError>;
+    type Error = SdkError<PutMetricDataError>;
     type Response = ();
 
     fn is_retriable_error(&self, error: &Self::Error) -> bool {
         match error {
-            RusotoError::Service(PutMetricDataError::InternalServiceFault(_)) => true,
-            error => rusoto::is_retriable_error(error),
+            SdkError::ServiceError { err, raw: _ } => {
+                matches!(err.kind, PutMetricDataErrorKind::InternalServiceFault(_))
+            }
+            _ => false,
```

Comment:
Do we need to preserve the retry behavior that `rusoto::is_retriable_error` was giving us? I see it retried a number of other scenarios (like throttling).